gst-libav.yaml

```diff
@@ -0,0 +1,87 @@
+package:
+  name: gst-libav
+  version: "1.26.0"
+  epoch: 0
+  description: GStreamer plugin for using the encoders, decoders, muxers, and demuxers provided by FFmpeg.
+  copyright:
+    - license: LGPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ffmpeg-dev
+      - gst-plugins-base-dev
+      - gstreamer-dev
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GStreamer/gstreamer
+      tag: ${{package.version}}
+      expected-commit: d31ce8e5e1aacf8f5e5beabb5c81ce2e4da5c202
+
+  - working-directory: subprojects/gst-libav
+    uses: meson/configure
+    with:
+      opts: |
+        -Db_lto=true
+      output-dir: output
+
+  - working-directory: subprojects/gst-libav
+    uses: meson/compile
+
+  - working-directory: subprojects/gst-libav
+    uses: meson/install
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: GStreamer/gstreamer
+    use-tag: true
+    tag-filter-prefix: "1."
+
+test:
+  environment:
+    contents:
+      packages:
+        - git
+  pipeline:
+    - uses: test/tw/ldd-check
+    - name: Clone GStreamer repo and run tests
+      runs: |
+        git clone --depth 1 https://github.com/GStreamer/gstreamer.git
```

Comment:
We can use `git-checkout` as we did above to ensure the test environment is consistent

Reply:
If we use git-checkout in our test pipeline then that will fail on every update. The bot as of writing this, doesn't update the commit recursively hence we don't get test git-checkout commit updated on update. (ref: https://github.com/wolfi-dev/os/pull/48685) 

I think instead checking main and testing, we can consider doing the following: 

```bash
git clone --branch ${{package.version}} --depth 1 https://github.com/GStreamer/gstreamer.git
```